tests/safety/test_toyota.py

```diff
@@ -0,0 +1,185 @@
+#!/usr/bin/env python2
+import unittest
+import numpy as np
+import libpandasafety_py
+
+MAX_RATE_UP = 10
+MAX_RATE_DOWN = 25
+MAX_TORQUE = 1500
+
+MAX_ACCEL = 1500
+MIN_ACCEL = -3000
+
+MAX_RT_DELTA = 375
+RT_INTERVAL = 250000
```

Comment:
Can we import those parameters from the safety_xxx.h file?

Reply:
The test should duplicate them, it's a better test that way